libMBIN/Source/NMS/GameComponents/GcMessageFiendCrime.cs

```diff
@@ -5,10 +5,10 @@
 
 namespace libMBIN.NMS.GameComponents
 {
-    [NMS(Size = 0x20, GUID = 0x5873ED86AF6E466E, NameHash = 0x37D134945C6DC0E)]
+    [NMS(Size = 0x20, GUID = 0x5873ED86AF6E466E, NameHash = 0x037D134945C6DC0E)]
```

Comment:
I was actually removing these leading 0's intentionally since they aren't needed...

Reply:
They screw up with automatically comparing files when the 0 is not there

Reply:
I mean, you should probably be comparing the values, not the string representation.
But anyway, It's not a major thing, but you'll find more instances of this because I have been intentionally formatting them without the leading 0.